client/src/constants/messages.js

```diff
@@ -21,6 +20,10 @@ export const TICKETS =
   'This page allows you to submit a help request via an RT Ticket.';
 export const UI =
   'This hidden page allows developers to review UI components in isolation.';
+
+/**
+ * Error Message Texts
```

Comment:
Update based on [idea to rename so as to not mislead users away from using markup](https://github.com/TACC/Core-Portal/pull/445/files/e88fd47840943de48a795a9e3b4084b19f0d041f#r690487209).

```suggestion
 * Error Messages
```